nextcord/application_command.py

```diff
@@ -1653,12 +1653,26 @@ class SlashCommandMixin(CallbackMixin):
         _description: Optional[str]
         command_ids: Dict[Optional[int], int]
         qualified_name: str
+        _children: Dict[str, SlashApplicationSubcommand]
 
     def __init__(self, callback: Optional[Callable], parent_cog: Optional[ClientCog]):
         CallbackMixin.__init__(self, callback=callback, parent_cog=parent_cog)
         self.options: Dict[str, SlashCommandOption] = {}
-        self.children: Dict[str, SlashApplicationSubcommand] = {}
         self._parsed_docstring: Optional[Dict[str, Any]] = None
+        self._children: Dict[str, SlashApplicationSubcommand] = {}
+
+    @property
+    def children(self) -> Dict[str, SlashApplicationSubcommand]:
+        """Returns the sub-commands and sub-command groups of the command.
+
+        .. versionupdated:: 2.3
+
+            ``.children`` is now a read-only property.
+        """
+        if self._children is not None:
```

Comment:
When is `_children` None? It's only a dictionary.

Reply:
Fixed in https://github.com/nextcord/nextcord/pull/851/commits/f81e013f726be00b94c8f97321ff0ddd0876e374